src/components/Card/Card.tsx

```diff
@@ -14,7 +14,7 @@ const Card: React.FC<CardProps> = ({ imgSrc, title, linkPath, status }) => {
   return (
     <div className={`${styles.card} ${status ? styles.cardWithStatus : ""}`}>
       {status && <TaskStatus status={status}></TaskStatus>}
-      <img src={imgSrc} alt="" />
+      <img src={imgSrc} alt={title + " image"} />
```

Comment:
alt="" w tym przypadku był pozostawiony celowo pusty, ponieważ wtedy oprogramowania dla niewidomych pomijają taki obrazek. Stosuje się to wtedy, kiedy obrazek nie wnosi swoją treścią nic poza "estetycznością".

![obraz](https://github.com/szymonkadas/Lesson4Future2.0-Setup/assets/95475451/283a6336-88d1-47c1-8c77-a64ac8fdb7f1)
https://www.w3.org/WAI/test-evaluate/preliminary/#images